drivers/include/at.h

```diff
@@ -186,19 +209,49 @@ typedef struct {
 #endif
 } at_dev_t;
 
+/**
+ * @brief AT device initialization parameters
+*/
+typedef struct {
+    uart_t uart;            /**< UART device where the AT device is attached */
+    uint32_t baudrate;      /**< UART device baudrate */
+    char *rx_buf;           /**< UART rx buffer */
+    size_t rx_buf_size;     /**< UART rx buffer size */
+    /**
+     * Response parsing buffer - used for classifying DCE responses and holding
+     * detailed error information. Must be at least 16 bytes.
+     * If you don't care about URCs (MODULE_AT_URC is undefined) this must only
+     * be large enough to hold responses like `OK`, `ERROR` or `+CME ERROR: <err_code>`.
+     * Otherwise adapt its size to the maximum length of the URCs you are expecting
+     * and actually care about. */
+    char *rp_buf;
+    size_t rp_buf_size;     /**< response parsing buffer size */
+} at_dev_init_t;
+
+/**
+ * @brief Get extended error information of the last command sent.
+ *
+ * If a previous at_* method returned with -AT_ERR_EXTENDED, you can retrieve
+ * a pointer to the error string with this.
+ *
+ * @param[in] dev device to operate on
+ *
+ * @retval `\0`-terminated string containing the error value.
```

Comment:
```suggestion
 * @retval 0-terminated string containing the error value.
```
Doxygen still doesn't like that.
We could also drop the '0-terminated' entirely since it's a bit redundant, strings in C are always expected to be zero-terminated.